.github/workflows/test.yml

```diff
@@ -0,0 +1,135 @@
+name: Test
+on:
+  push:
+    branches: [develop]
+  pull_request:
+    branches: [develop]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: yarn
+        cache-dependency-path: "**/yarn.lock"
+    - run: yarn install
+    - run: yarn test:health
+    - run: yarn flow
+    - run: yarn typescript
+    - run: yarn lint
+    - run: yarn test:unit
+
+  build:
+    runs-on: ubuntu-18.04
```

Comment:
do we still need to use ubuntu 18.04 for the build? Assume the latest would work fine for node12 also, or?

Reply:
I just used what was already there in the `develop` branch, since I'm unfamiliar with karma/your build tools. https://github.com/trezor/connect/blob/6368fcda5f6abe7f31b5de8c9dea51a5509866a3/.github/workflows/test_api_karma.yml#L11

However, I tried upgrading it to `ubuntu-latest` in 86c0e578ca1989a857504638c1c5244f32cf0817 and it seems to work.